src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -198,9 +223,36 @@ object TelemetryTypes extends Enumeration {
   val SPARK_STREAMING_END: Value = Value("spark_streaming_end")
   val SPARK_EGRESS: Value = Value("spark_egress")
   val SPARK_CLIENT_INFO: Value = Value("spark_client_info")
+  val SPARK_QUERY_RESULT: Value = Value("spark_query_result")
   val SPARK_PUSHDOWN_FAIL: Value = Value("spark_pushdown_fail")
 }
 
+object TelemetryConstValues {
+  val OPERATION_READ = "read"
+  val OPERATION_WRITE = "write"
+  val STATUS_SUCCESS = "success"
+  val STATUS_FAIL = "fail"
+}
+
+object TelemetryQueryResultFields {
+  // Spark connector version
+  val SPARK_CONNECTOR_VERSION: String = "spark_connector_version"
+  // The unsupported operation for pushdown
+  val OPERATION: String = "operation"
+  // Query statement
+  val QUERY: String = "query"
+  // query ID if available
+  val QUERY_ID: String = "queryid"
+  // Query result: fail vs success
+  val QUERY_RESULT: String = "result"
+  // query execution time
+  val QUERY_ELAPSE: String = "elapse"
+  // The error message for the exception
+  val EXCEPTION_MESSAGE: String = "message"
```

Comment:
exception_message

Reply:
will update

Reply:
After considering it again, I change it back to "message".
The reason is that we have already sent Exception with OOB telemetry message on executors. I think we'd better  use the same field name for exception. Otherwise, it may confusing when querying the message from snowhouse.